src/main/app/src/translations/fi.json

```diff
@@ -605,7 +605,9 @@
         "yhteystiedotInfo": "Anna oppilaitokselle tarvittavat yhteystiedot",
         "oppilaitoksenTila": "Oppilaitoksen tila",
         "oppilaitoksenTeemakuva": "Oppilaitoksen teemakuva",
-        "logo": "Oppilaitoksen logo"
+        "logo": "Oppilaitoksen logo",
+        "eiMuokkausOikeutta": "Käyttäjälläsi ei ole oikeuksia muokata tätä oppilaitosta."
```

Comment:
Olikse speksissäkin Käyttäjälläsi?
Kai se oikeemmin on Käyttäjätunnuksellasi

Sama tos alempana

Reply:
Tämä puuttui oppilaitokselta ja oppilaitoksen osalta. Kaikille muille oli laitettu tuohon tyyliin. Nämä tulee joka tapauksessa koutaankin nykyään käännöspalvelusta, että ei varmaan kannata alkaa tänne näitä korjailemaan.